.github/workflows/updateversion.yml

```diff
@@ -14,14 +23,13 @@ jobs:
         ( github.event.review.author_association == 'OWNER' ||
           github.event.review.author_association == 'MEMBER'
         ) &&
-        ! contains( github.event.review.body, '@github-actions noop' )
+        ! contains( github.event.review.body, '@github-actions :x:' )
       ) ||
-      ( github.event_name == 'issue_comment' &&
-        github.event.issue.pull_request &&
-        ( github.event.comment.author_association == 'OWNER' ||
-          github.event.comment.author_association == 'MEMBER'
+      ( github.event_name == 'pull_request_review_comment' &&
+        ( github.event.review.author_association == 'OWNER' ||
+          github.event.review.author_association == 'MEMBER'
         ) &&
-        endsWith( github.event.comment.body, '@github-actions update version' )
+        contains( github.event.review.body, '@github-actions :ship:' )
```

Comment:
@ebbit1q Any reason you used `endsWith` here over the more flexible `contains` as in the other condition?